### ISSUE TITLE
Navigation buttons in journals

### ISSUE THREAD

**Issue by hmarthinsen** [foundryvtt/foundryvtt#8323]
### User Experience

It would be really useful to have back/forward navigation buttons in the journal UI, especially for journals with many pages and hyperlinks between them. As it is now, I have to remember where I came from when I click a link, and then manually navigate back. One example where this is cumbersome is the "GM Screen" journal entry in the Pathfinder 2 game system.

**Comment by wake42** [comment]
I agree fully with this case. I'm currently thinking about how to organize a large journal with multiple pages and links between these pages, and not being able to return to the page you came from is a big problem I think. When the link comes from another journal it is not a problem, but internal links are problematic.
